.travis.yml

```diff
@@ -1,24 +1,24 @@
 sudo: required
 dist: trusty
-# Force travis to use its minimal image with default Python settings
 language: generic
-compiler:
-  - gcc
-
+notifications:
+  email:
+    on_success: change
+    on_failure: always
 env:
   global:
-    - CATKIN_WS=~/catkin_ws
-    - CATKIN_WS_SRC=${CATKIN_WS}/src
-    - CATKIN_WS_UNDERLAY=~/catkin_ws_underlay
-    - CATKIN_WS_UNDERLAY_SRC=${CATKIN_WS_UNDERLAY}/src
-    - CI_ROS_DISTRO="indigo"
-    - DEFAULT_ROSINSTALL="https://raw.githubusercontent.com/ipa320/care-o-bot/${CI_ROS_DISTRO}_dev/.travis.rosinstall"
+    - UPSTREAM_WORKSPACE=file
+    - BEFORE_SCRIPT:'./.travis.xacro_test.sh'
```

Comment:
Then, is the [docu](https://github.com/ros-industrial/industrial_ci/blob/master/doc/index.rst#run-pre-install-custom-commands) incorrect? There it is stated as `- BEFORE_SCRIPT:'./your_custom_PREprocess.sh'`.